lib/Yam/Agama/Pom/GrubMenuAgamaPage.pm

```diff
@@ -26,4 +26,10 @@ sub expect_is_shown {
 
 sub edit_current_entry { shift->{grub_menu_base}->edit_current_entry() }
 
+sub boot_from_hd {
```

Comment:
let's put all methods that are not simple by-passing on top, so this one should go before the previous one, it is more readable.